modules/tinymce/src/plugins/codesample/test/ts/webdriver/CodeSampleCopyAndPasteTest.ts

```diff
@@ -0,0 +1,82 @@
+import { Keys, RealClipboard } from '@ephox/agar';
+import { beforeEach, describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/codesample/Plugin';
+
+import * as TestUtils from '../module/CodeSampleTestUtils';
+
+describe('webdriver.tinymce.plugins.codesample.CodeSampleCopyAndPasteTest', () => {
```

Comment:
We also need a test here that does just a plain `setContent()` call to make sure it works correctly as well. As such I'd also suggest renaming this to `CodeSampleInsertTest` or similar...